op-e2e/system_test.go

```diff
@@ -840,3 +841,122 @@ func TestWithdrawals(t *testing.T) {
+
+	// Transactor Account
+	ethPrivKey, err := sys.wallet.PrivateKey(accounts.Account{
+		URL: accounts.URL{
+			Path: transactorHDPath,
+		},
+	})
+	require.Nil(t, err)
+	fromAddr := crypto.PubkeyToAddress(ethPrivKey.PublicKey)
+
+	// Find gaspriceoracle contract
+	gpoContract, err := bindings.NewGasPriceOracle(common.HexToAddress(predeploys.OVM_GasPriceOracle), l2Seq)
+	require.Nil(t, err)
+
+	// GPO signer
+	l2opts, err := bind.NewKeyedTransactorWithChainID(ethPrivKey, cfg.L2ChainID)
+	require.Nil(t, err)
+
+	// Update overhead
+	tx, err := gpoContract.SetOverhead(l2opts, big.NewInt(2100))
+	require.Nil(t, err, "sending overhead update tx")
+
+	receipt, err := waitForTransaction(tx.Hash(), l2Verif, 10*time.Duration(cfg.L1BlockTime)*time.Second)
+	require.Nil(t, err, "waiting for overhead update tx")
+	require.Equal(t, receipt.Status, types.ReceiptStatusSuccessful, "transaction failed")
+
+	// Update decimals
+	tx, err = gpoContract.SetDecimals(l2opts, big.NewInt(6))
+	require.Nil(t, err, "sending gpo update tx")
+
+	receipt, err = waitForTransaction(tx.Hash(), l2Verif, 10*time.Duration(cfg.L1BlockTime)*time.Second)
+	require.Nil(t, err, "waiting for gpo decimals update tx")
+	require.Equal(t, receipt.Status, types.ReceiptStatusSuccessful, "transaction failed")
+
+	// Update scalar
+	tx, err = gpoContract.SetScalar(l2opts, big.NewInt(1_000_000))
+	require.Nil(t, err, "sending gpo update tx")
+
+	receipt, err = waitForTransaction(tx.Hash(), l2Verif, 10*time.Duration(cfg.L1BlockTime)*time.Second)
+	require.Nil(t, err, "waiting for gpo scalar update tx")
+	require.Equal(t, receipt.Status, types.ReceiptStatusSuccessful, "transaction failed")
+
+	overhead, err := gpoContract.Overhead(&bind.CallOpts{})
+	require.Nil(t, err, "reading gpo overhead")
+	decimals, err := gpoContract.Decimals(&bind.CallOpts{})
+	require.Nil(t, err, "reading gpo decimals")
+	scalar, err := gpoContract.Scalar(&bind.CallOpts{})
+	require.Nil(t, err, "reading gpo scalar")
+
+	require.Equal(t, overhead.Uint64(), uint64(2100), "wrong gpo overhead")
+	require.Equal(t, decimals.Uint64(), uint64(6), "wrong gpo decimals")
+	require.Equal(t, scalar.Uint64(), uint64(1_000_000), "wrong gpo scalar")
+
+	// Fee Recipient
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
+	defer cancel()
+	feeStartBalance, err := l2Seq.BalanceAt(ctx, cfg.FeeRecipient, nil)
+	require.Nil(t, err)
+
+	// Simple transfer from signer to random account
+	ctx, cancel = context.WithTimeout(context.Background(), 1*time.Second)
+	defer cancel()
+	startBalance, err := l2Verif.BalanceAt(ctx, fromAddr, nil)
+	require.Nil(t, err)
+
+	toAddr := common.Address{0xff, 0xff}
+	transferAmount := big.NewInt(1_000_000_000)
+	tx = types.MustSignNewTx(ethPrivKey, types.LatestSignerForChainID(cfg.L2ChainID), &types.DynamicFeeTx{
+		ChainID:   cfg.L2ChainID,
+		Nonce:     3, // Already have deposit
+		To:        &toAddr,
+		Value:     transferAmount,
+		GasTipCap: big.NewInt(0),
```

Comment:
This fails with any `GasTipCap` other than `0`. See: https://github.com/ethereum-optimism/reference-optimistic-geth/pull/26/files#r896147368

Reply:
nevermind, I was missing the coinbase reward.